Tests/mocks/devicesettings.h

```diff
@@ -980,111 +995,219 @@ class Host {
         return impl->getDefaultAudioPortName();
     }
 };
-
 }
-
-namespace device {
-class FrontPanelIndicator;
-class FrontPanelIndicatorImpl {
-public:
-    virtual ~FrontPanelIndicatorImpl() = default;
-    virtual void setState(const bool bState) const = 0;
-    virtual const std::string & getName() const = 0;
-    virtual void setBrightness(const int &brightness, const bool toPersist = true);
-    virtual int getBrightness();
-};
-
-class FrontPanelIndicator {
-public:
-    FrontPanelIndicatorImpl* impl;
-    class Color{
-        public:
-            virtual ~Color() {};
-
-            static Color & getInstance(int id)
-            {
-                static Color instance;
-                return instance;
-            }
-            static Color & getInstance(const std::string &name)
-            {
-                return getInstance(0);
-            }
-    };
-    static FrontPanelIndicator& getInstance(const std::string&)
-    {
-        static FrontPanelIndicator instance;
-        return instance;
-    }
-
-    void setState(const bool bState) const
-    {
-        return impl->setState(bState);
-    }
-
-    const std::string & getName() const
-    {
-        return impl->getName();
-    }
-    void setBrightness(const int &brightness, const bool toPersist = true)
-    {
-        return impl->setBrightness(brightness, toPersist);
-    }
-    int getBrightness()
-    {
-        return impl->getBrightness();
-    }
-    void setColor(const Color & newColor,bool toPersist = true)
-    {
-    }
-    void setColor(const uint32_t color,const bool toPersist = true)
-    {
-    }
-};
-
-class FrontPanelTextDisplay
-{
-public:
-    static const int kModeClock12Hr = dsFPD_TIME_12_HOUR;
-    static const int kModeClock24Hr = dsFPD_TIME_24_HOUR;
-
-    virtual ~FrontPanelTextDisplay() = default;
-
-    int getCurrentTimeFormat()
-    {
-        return kModeClock12Hr;
-    }
-    void setTimeFormat(const int iTimeFormat)
-    {
-    }
-};
-
-class FrontPanelConfig;
-class FrontPanelConfigImpl {
-public:
-    virtual ~FrontPanelConfigImpl() = default;
-    virtual List<FrontPanelIndicator> getIndicators();
-};
-
-class FrontPanelConfig {
-public:
-    FrontPanelConfigImpl* impl;
-
-    static FrontPanelConfig& getInstance()
-    {
-        static FrontPanelConfig instance;
-        return instance;
-    }
-    List<FrontPanelIndicator> getIndicators()
-    {
-        return impl->getIndicators();
-    }
-    FrontPanelTextDisplay& getTextDisplay(const std::string &name)
-    {
-        static FrontPanelTextDisplay instance;
-        return instance;
-    }
-};
-
+namespace device{
```

Comment:
please remove whitespace changes

Reply:
@npoltorapavlo The changes for this have already been made, I don't know why they're not reflecting here and sating the file is out dated. (This is what it looks like when viewing in the files changed--there are no white space changes anymore.)


<img width="938" alt="update" src="https://user-images.githubusercontent.com/107510770/209158638-4fe72c6b-ed94-4ff7-a4b9-215471778681.PNG">